### PR TITLE
MINOR: Update Streams quickstart to create output topic with compaction enabled

### PR DESCRIPTION
**Pull request by mjsax**

### CODE REVIEW

**Comment by mjsax** [comment]
Call for review @guozhangwang @bbejeck @dguy @miguno

File: docs/streams/quickstart.html [hunk 1]
```diff
@@ -130,11 +130,13 @@
...
 Created topic "streams-plaintext-input".
 
+# note: we create the output topic with compaction enabled because the output stream is a KTable
 &gt; bin/kafka-topics.sh --create \
     --zookeeper localhost:2181 \
     --replication-factor 1 \
     --partitions 1 \
-    --topic streams-wordcount-output
+    --topic streams-wordcount-output \
+    --config cleanup.policy=compact
```
Comment (guozhangwang) [inline docs/streams/quickstart.html:143]: actually, how about setting `cleanup.policy=delete,compact`?

File: docs/streams/quickstart.html [hunk 2]
```diff
@@ -130,11 +130,13 @@
     --topic streams-plaintext-input
 Created topic "streams-plaintext-input".
 
+# note: we create the output topic with compaction enabled because the output stream is a KTable
```
Comment (guozhangwang) [inline docs/streams/quickstart.html]: Users may not know about KTable yet here, maybe we can say "output stream is a changelog stream, see below (ref) for details".

**Review by guozhangwang — COMMENTED** [review]
I'm wondering if we should do the same for tutorials as well?

File: docs/streams/quickstart.html (hunk 1 above)
Comment (mjsax) [reply docs/streams/quickstart.html:143]: Don't think `delete,compact` would be correct.

**Comment by mjsax** [comment]
Updated

**Review by miguno — APPROVED** [review]

**Comment by guozhangwang** [comment]
LGTM. Merged to trunk.

File: docs/streams/quickstart.html (hunk 1 above)
Comment (guozhangwang) [reply docs/streams/quickstart.html:143]: Okay.

**Pull request closed without merging**